server/controllers/authController.js

```diff
@@ -29,6 +29,13 @@ exports.signUp = async (req, res) => {
         password: password,
     });
 
+    const message = `Dear ${newusername},\nWelcome to Judicature!\n\nWe're thrilled to have you join our community. At Judicature, we understand the importance of securing and verifying your documents. That's why we've created a platform where you can safely store your documents and have them verified by the government with ease.\n\nGetting Started:\n1. Upload Your Documents: Start by uploading the documents you wish to secure and verify.\n2. Request Verification: Once uploaded, you can request government verification for your documents.\n3. Share & Access: After verification, you can share your documents with trusted individuals or access them anytime, anywhere.\nRemember, your trust is our top priority. We're committed to providing you with a seamless experience and ensuring the utmost security for your documents.\n\nIf you have any questions or need assistance, please don't hesitate to reach out to our support team at support@judicature.com.\n\nThank you for choosing Judicature. We look forward to serving you!\nWarm regards,\nThe Judicature Team`;
+    await sendEmail({

```

Comment:
Add a check, in which if `process.env.NODE_ENV` is not `prod`, mail shouldn't be getting sent, and I would suggest adding this check in forgot password, if its not on prod, forgot password link should be consoled only